src/openrct2/core/Path.hpp

```diff
@@ -23,6 +23,7 @@ namespace Path
 {
     utf8 * Append(utf8 * buffer, size_t bufferSize, const utf8 * src);
     std::string Combine(const std::string &a, const std::string &b);
+    std::string Combine(const std::string &a, const std::string &b, const std::string &c);
```

Comment:
For the same reason you would only have `int add(int a, int b)` and no `int add(int a, int b, int c)`, `int add(int a, int b, int c, int d)` and so on…